test cleanup

nec4 opened the pull request:
Development:
 - [x] Implement feature / fix bug
 - [ ] Add documentation
 - [ ] Add tests
 
 Checks:
 - [x] Run `nosetests`
 - [x] Check pep8 compliance

Hello all. This PR is in response to #198 and addresses some intermittent test failures. In many places, I have raised the precisions of variables and adjusted certain tolerances. Two major fixes stand out:

1. For GPU tests, radial basis functions are now properly instanced so that the tests run properly.
2. In `test_simulation.test_saving_numpy_coordinates()`, the temporary filelist is now sorted (previously it was unsorted and this could lead to the loaded array being backwards). This does not affect the ability of simulations to be saved or loaded properly - it was just an issue with the test itself (and I think it might depend on how the `os.listdir()` sorts things by default).

All tests now pass for me and fave been PEP8 formatted. No changes to the main code have been made.

Commit:
test cleanup

Commit:
pep8

nec4 commented:
Once this is merged, I will address #199 and #197.

nec4 commented:
Yup! It's ready for review.

brookehus commented on cgnet/tests/test_molecule_dataset.py:
```diff
@@ -157,48 +160,48 @@ def test_multi_molecule_collate():
+
+    # get tensors output from multi_molecule_collate()
+    coords, forces, embeddings = multi_molecule_collate(data)
+    print(coords.size())
```
should this be here?

nec4 replied:
Nope - removed!

brookehus commented on cgnet/tests/test_simulation.py:
```diff
@@ -509,19 +511,20 @@ def test_saving_numpy_coordinates():
+    save_interval = 1  # np.random.choice([2, 3])
+
+    n_expected_files = int(sim_length / npy_interval)
+    print(n_expected_files)
```
should this be here?

nec4 replied:
Nope - removed!

brookehus commented on cgnet/tests/test_simulation.py:
```diff
@@ -533,8 +536,9 @@ def test_saving_numpy_coordinates():
-
+        file_list = sorted(os.listdir(tmp))
+        print(len(file_list))
+        print(traj)
```
should these print statements be here?

nec4 replied:
Nope - removed!

brookehus commented on cgnet/tests/test_simulation.py:
```diff
@@ -589,7 +592,8 @@ def test_saving_all_quantities():
 
-        assert len(file_list) == n_expected_files * 4  # coords, forces, pot, ke
+        assert len(file_list) == n_expected_files * \
+            4  # coords, forces, pot, ke
```
i would keep this the old way for readability

nec4 replied:
Good catch! Agreed.

brookehus reviewed (changes requested):
all looks good, thanks for doing that! just curious about a few print statements and one aesthetic thing

Commit:
print removal/refactor

nec4 commented:
Thanks for taking a look! I removed those print statements and reverted the assert line back to its original state. I reran the tests and they all pass for me. Let me know if there is anything else that can be changed or improved.

brookehus reviewed (approved):
awesome thank you!
